src/kvstore/comm.h

```diff
@@ -22,29 +22,28 @@
  */
 #ifndef MXNET_KVSTORE_COMM_H_
 #define MXNET_KVSTORE_COMM_H_
+#define NVLINK_SUPPORT 4
 #include <dmlc/omp.h>
-#include <string>
 #include <algorithm>
-#include <utility>
 #include <limits>
-#include <vector>
-#include <tuple>
+#include <string>
 #include <thread>
-#include "mxnet/ndarray.h"
-#include "gradient_compression.h"
+#include <tuple>
+#include <utility>
+#include <vector>
 #include "../ndarray/ndarray_function.h"
 #include "../operator/tensor/sparse_retain-inl.h"
+#include "gradient_compression.h"
+#include "mxnet/ndarray.h"
 namespace mxnet {
 namespace kvstore {
 /**
  * \brief multiple device commmunication
  */
 class Comm {
  public:
-  Comm() {
-    pinned_ctx_ = Context::CPUPinned(0);
-  }
-  virtual ~Comm() { }
+  Comm() { pinned_ctx_ = Context::CPUPinned(0); }
```

Comment:
Why don't we initialize on the constructor initialization list? It's more efficient.